Package.swift

```diff
@@ -233,6 +233,16 @@ let package = Package(
         .headerSearchPath("../../.."),
       ]
     ),
+    .target(
+      name: "FirebaseCoreSwift",
```

Comment:
How about `FirebaseCoreInternalSwift`?  To be clear that these APIs are only for internal Firebase usage?